examples/app_commands/info.py

```diff
@@ -1,45 +1,43 @@
-import discord
-from discord.ext import commands
+# This example requires the 'members' privileged intent to use the Member converter.
 
-# imports
+import discord
 
-intents = discord.Intents(
-    guilds=True,
-    members=True,
-    messages=True,
-)
+intents = discord.Intents.default()
+intents.members = True
 
-bot = commands.Bot(
-    command_prefix="/",
-    description="An example to showcase how to extract info about users",
+bot = discord.Bot(
+    debug_guilds=[...],
+    description="An example to showcase how to extract info about users.",
     intents=intents,
 )
 
 
-@bot.slash_command(name="userinfo", description="gets the info of a user")
+@bot.slash_command(name="userinfo", description="Gets info about a user.")
```

Comment:
Why not typehint context here?

Reply:
Missed these 3 type hints somehow, fixed in the latest commit.